br/pkg/lightning/backend/external/writer_test.go

```diff
@@ -414,3 +415,39 @@ func TestWriterMultiFileStat(t *testing.T) {
 	require.EqualValues(t, "key01", summary.Min)
 	require.EqualValues(t, "key24", summary.Max)
 }
+
+func TestWriterSort(t *testing.T) {
```

Comment:
Use bench？

Reply:
I just want to make sure the data are sorted correctly.